js/speakers.js

```diff
@@ -0,0 +1,19 @@
+import speakersInfo from './speakersInfo.js';
+
+document.querySelector('#speakers__container').innerHTML = speakersInfo.map(
+  (info) => `
+    <div class="speaker">
+    <img src=${info.Image} alt="Speaker's face" class="speaker__img" />
+    <div class="speaker__body">
+      <h4 class="speaker__name">${info.name}</h4>
+      <span class="speaker__summary">
+        ${info.summary}
+      </span>
+      <span class="bar--primary"></span>
+      <p class="speaker__details">
+        ${info.details}
+      </p>
+    </div>
+  </div> `,
```

Comment:
Please kindly wrap individual event with an article tag according to the best practises. The article tag specifies independent, self-contained content and its semantic unlike `div`. 👍